django_netjsongraph/base/admin.py

```diff
@@ -23,6 +23,20 @@ def __init__(self, *args, **kwargs):
 class BaseAdmin(TimeStampedEditableAdmin):
     save_on_top = True
 
+    def get_extra_context(self, pk=None):
```

Comment:
Would give some wrong results then actually. Because the `pk` wouldn't be the topology id then.